README.md

```diff
@@ -21,8 +20,12 @@ var results = AuthorModel.find({name: "Bob"});
 console.log(results);
 ```
 
+This is a Mongoose Query. It needs a callback.
+
 ```js
-// Your answer...
+var results = AuthorModel.find({name: "Bob"}, results) => {
+console.log(results);
+}
```

Comment:
👍 paren after results needs to move to end of 28, but this is correct! also add `err` to callback